.gitignore

```diff
@@ -17,4 +17,8 @@ dist/
 .pytest_cache/
 
 # Other
-*.DS_Store
\ No newline at end of file
+*.DS_Store
```

Comment:
You should delete this changes if you want add this extensions to gitignore file, you can open new pr about this